### ISSUE TITLE
Issue with scripts loading wierd.

### ISSUE THREAD

**Issue by Claycorp** [stanhebben/MineTweaker3#180]
After loading the pack up some recipes are are just deleted and not replace, replaced but not deleted or just deleted. It seems to be fairly random at what is affected and how its affected.

Doing a /mt reload fixes everything but this isn't a viable solution when in a pack. This part confuses me the most as you would think that if its loading wrong it won't load properly after that.
 
I have no idea what is wrong and have even tested it in new files, split up, other launchers and nothing seems to fix it.

I have been using MineTweaker + ModTweaker for a while now and have never ran into this issue before. Any help would be great!

Script
http://pastebin.com/rg3WvcS1

**Comment by skaviouz** [comment]
i'd suggest including the minetweaker.log; you can find the log in the root folder.

**Comment by Claycorp** [comment]
http://pastebin.com/2hzdZSkE There it is even though its not an error with the scripts its with the mod. On top of that things don't sync properly on the server.

**Comment by go371211** [comment]
I got the same issue. It's really bothering me because what recipes are affected seems completely random.

**Comment by jaredlll08** [comment]
Can this be reproduced without minetweaker recipe maker?

**Comment by Claycorp** [comment]
This issue is 1 year and 3 months old... There's no way in hell anyone can reproduce it anymore. Also MTRM doesn't do anything to MT so it's an issue with MT itself.

**Comment by Lanse505** [comment]
@Claycorp
Also it's an open issue so @jaredlll08 can try to respond to it whenever he wants 
(since he's the developer), Also MTRM "does things" with MT since it hooks into Minetweaker to be able to know how to write the script?

I'd recommend not coming off so.... hostile towards one of the authors and the current maintainer.
He's just trying to help and work through the issue, no need to be so hostile in your messages.

**Comment by Claycorp** [comment]
@Lanse505 Will you stop replying to my issues? You have NO input to the actual problem and are just trying to start shit.
He can respond to it and I informed him that there is no way to reproduce it this far down the road because it's been way too long and too much has changed,
Also please check who made MTRM before you talk from your ass. Thanks.

**Comment by dries007** [comment]
@Lanse505 Please do some research before you make comments like that.
The only thing MTRM does is write a script file to minetweaker's specs. It can run without it installed, but since the script wouldn't load that would kinda make it pointless.

This is the only line in the entire codebase that actually involves Minetweaker:

``` java
Class.forName("minetweaker.MineTweakerImplementationAPI").getDeclaredMethod("reload").invoke(null);
```

[And it gets try-catch'd to avoid anything going wrong.](https://github.com/DoubleDoorDevelopment/MineTweakerRecipeMaker/blob/master/src/main/java/net/doubledoordev/mtrm/network/MessageSend.java#L191-L198)

ALSO: I hope you are aware that we can see WHO spams the emotes.
![](https://dries007.net/screenshots/2016-08-22_19-26-52.png)

**Comment by Lanse505** [comment]
Yea?

I downvoted my own comment and upvoted yours cause I was wrong and you were right...
Is it bad that I decided to show that I myself think I was wrong?

**Comment by dries007** [comment]
I thought it was meant to make clay look evil. You can actually also edit your previous comment, that's usually what I do. Since Github supports markdown, you can use a double tilde `~` to cross something out if you wanted to. ~~like this!~~

Also, I'm pretty sure this issue can be closed by now, it was about Minetweaker 1.7, and I don't think it's present in later versions. If it pops up again, we can re-open it.

**Comment by jaredlll08** [comment]
@Claycorp if you look, there are issues that were left idle for longer, yet the person who made it was happy to help and provide the files. So either help, or stop complaining.

Also I asked @Lanse505 to send a reply because it was him sending something nice, or me using every word in a salty sailor's dictionary to describe you.

I'm locking this issue, it is more than obvious that you don't want to help, and I don't want to have to deal with you telling me that my update schedules are non existent or that I should spend my whole waking life working on issues, and should reply and fix issues within 5 minutes, even though this issue was made before I was working on minetweaker.
